installer/conf/omsagent.conf

```diff
@@ -57,9 +57,9 @@
   buffer_queue_limit 10
   buffer_queue_full_action drop_oldest_chunk
   flush_interval 60s
-  retry_limit 10
+  retry_limit 6
```

Comment:
does this do exponential backoff?

Reply:
exponential backoff is implemented in fluentd by default.